src/badges.js

```diff
@@ -527,7 +514,7 @@ function validateTemplate(template, locations) {
 							if (['name', 'type', 'label', 'description', 'defaultValue'].includes(key) && (typeof parameter[key] !== 'string' || !parameter[key])) {
 								invalidParam.push(`template paramater '${detail}[${i}].${key}' must be a string with a value`);
 							}
-							const allowedTypes = ['array', 'text', 'color', 'url', 'integer'];
+							const allowedTypes = ['array', 'string', 'color', 'url', 'integer', 'decimal', 'boolean', 'checkbox', 'toggle'];
```

Comment:
Just noticing this, but are boolean, checkbox and toggle essentially the same?
We should find out what is actually supported.